npm/src/updateconsignmentstatus/index.ts

```diff
@@ -18,23 +19,28 @@ export class UpdateConsignmentStatus {
 
   async markConsignmentStatusAsCompleted(
     uploadFilesInfo: FileUploadInfo
-  ): Promise<number> {
-    const variables: MarkUploadAsCompletedMutationVariables = {
-      consignmentId: uploadFilesInfo.consignmentId
-    }
+  ): Promise<number | void> {
+    try {
+      const variables: MarkUploadAsCompletedMutationVariables = {
+        consignmentId: uploadFilesInfo.consignmentId
+      }
 
-    const result: FetchResult<MarkUploadAsCompletedMutation> =
-      await this.client.mutation(MarkUploadAsCompleted, variables)
+      const result: FetchResult<MarkUploadAsCompletedMutation> =
+        await this.client.mutation(MarkUploadAsCompleted, variables)
 
-    if (!result.data || !result.data.markUploadAsCompleted || result.errors) {
-      const errorMessage: string = result.errors
-        ? result.errors.toString()
-        : "no data"
-      throw Error(
-        `Marking the Consignment Status as "Completed" failed: ${errorMessage}`
+      if (!result.data || !result.data.markUploadAsCompleted || result.errors) {
+        const errorMessage: string = result.errors
+          ? result.errors.toString()
+          : "no data"
+        throw Error(errorMessage)
+      } else {
+        return result.data.markUploadAsCompleted
+      }
+    } catch (e) {
+      handleUploadError(
```

Comment:
Can this try/catch be deleted now that upload/index.ts calls `handleUploadError`?

Reply:
🤦  ah, I thought I deleted it. Thanks